### ISSUE TITLE
Typescript - Expected X arguments, but got Y

### ISSUE THREAD

**Issue by igorsheg-wix** [cristianbote/goober#91]
Hi there! Awesome work with this library.

I'm using `preact-custom-element` with TypeScript,  and it's throwing an error since when trying to use props inside a `styled` more than once

<img width="572" alt="image" src="https://user-images.githubusercontent.com/26436913/72681404-d97f6b80-3acb-11ea-92bd-4aed36e66fbe.png">

I'm getting: 

```
Expected 1-2 arguments, but got 3.ts(2554)
```
The examples don't seem to have to specify this, so is this something with my code or the library?

Here are my relevant dependencies:
```
 "goober": "^1.6.3",
 "preact": "^10.2.1",
"typescript": "^3.7.5"
```

**Comment by cristianbote** [comment]
Hey @igorsheg-wix, thank you so much for opening this one and I appreciate you checking out goober 😄 I confirm this is an issue and I'm looking into it.

**Comment by cristianbote** [comment]
Hey @igorsheg-wix the fix is in `goober@1.6.4`. Let me know if there are other issues.

Thank you for reporting this, I really appreciate it!

**Comment by igorsheg-wix** [comment]
@cristianbote Fixed! thanks for the quick response! ❤️
